_routes/post_interactions.md

```diff
@@ -39,8 +39,11 @@ Status: 204 No Content
 
 | Status code | Error reason                               |
 | ----------- | ------------------------------------------ |
+| 404         | Post with provided guid could not be found |
 | 409         | Post has already been reported             |
+| 422         | Request invalid, such as a missing reason  |
 
+>>>>>>> Add error docs to post interactions and undo adding a post body to mute
```

Comment:
Dat was a merge going wrong. :)

Reply:
dammit

Reply:
I wholeheartedly blame sublime for this one
![Screenshot from 2020-02-02 18-58-15](https://user-images.githubusercontent.com/141294/73612674-08c6ca00-45ee-11ea-9593-3ca9621111a5.png)

Reply:
Wow.. that's something, fore sure.